Bottom: c6a9952f8d4351d520eac3ffc21bb04df877846e
Top:    8be921caba590044cb53424cba0e659eb4340264
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:36:48 +0000

Refresh of continue-tinkering-with-3d

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index cc2fa1d..d08a189 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -445,11 +445,11 @@ object3d_from_board_outline (void)
                                       -(vertices[next_i_around_ct]->x - vertices[i]->x), 0.);
 
           /* Assign the appropriate vertex geometric data to each edge end */
-          ODATA (edges[              i]) = vertices[0 * npoints + i];
-          DDATA (edges[              i]) = vertices[0 * npoints + next_i_around_ct];
+          ODATA (edges[              i]) = vertices[i];
+          DDATA (edges[              i]) = vertices[next_i_around_ct];
           ODATA (edges[1 * npoints + i]) = vertices[1 * npoints + i];
           DDATA (edges[1 * npoints + i]) = vertices[1 * npoints + next_i_around_ct];
-          ODATA (edges[2 * npoints + i]) = vertices[0 * npoints + i];
+          ODATA (edges[2 * npoints + i]) = vertices[i];
           DDATA (edges[2 * npoints + i]) = vertices[1 * npoints + i];
           LDATA (edges[              i]) = faces[i];
           RDATA (edges[              i]) = faces[npoints];
